pdns/recursordist/docs/lua-scripting/dq.rst

```diff
@@ -223,6 +223,20 @@ The DNSQuestion object contains at least the following fields:
 
       The packet-cache tag set via :func:`gettag`, or 0 if it has not been set.
 
+  .. attribute:: DNSQuestion.queryTime
+
+     .. versionadded:: 4.8.0
+
+     The time the query was received
+
+     .. attribute:: DNSQuestion.queryTime.tv_sec
+
+        The number of seconds since the Unix epoch.
+
+     .. attribute:: DNSQuestion.queryTime.tv_usec
+
+        The number of microseconds.
```

Comment:
Perhaps a clarification that this is the number of microseconds on top of the number of seconds `tv_sec` (IIRC)?